python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -367,17 +367,34 @@ def computeCorrection(self, kappa, svar, tvar, preConvArr=None):
         The maximum correction factor converges to `sqrt(tvar/svar)` towards high frequencies.
         This should be a plausible value.
         """
+        kSum = np.sum(kappa)
         kappa = self.padCenterOriginArray(kappa, self.freqSpaceShape)
         kft = np.fft.fft2(kappa)
-        kft2 = np.real(np.conj(kft) * kft)
-        if preConvArr is None:
-            denom = svar + tvar * kft2
-        else:
+        kftAbsSq = np.real(np.conj(kft) * kft)
+        # If there is no pre-convolution kernel, use placeholder scalars
+        preSum = 1.
+        preAbsSq = 1.
+        if preConvArr is not None:
```

Comment:
I've been scolded for using `is not None` which can be a bit hard to read. Could you change this structure back to the if/else setting the `preSum`, `preAbsSq` in the `if is None` statement.